src/shell/shell_cmds.cpp

```diff
@@ -1468,7 +1468,7 @@ void DOS_Shell::CMD_SET(char * args) {
 	char * pcheck = args;
 	while ( *pcheck && (*pcheck == ' ' || *pcheck == '\t')) pcheck++;
 	if (*pcheck && strlen(pcheck) > 3 && (strncasecmp(pcheck, "/p ", 3) == 0)) {
-		E_Exit("Set /P is not supported. Use Choice!");
+		WriteOut(MSG_Get("SHELL_CMD_SET_OPTION_P_UNSUPPORTED"));
```

Comment:
Shouldn't we return from the function here?

Reply:
Kinda works out fine as from here we just fall through the rest, but it's clear too add a return. Done.